test/e2e-v2/cases/meter/meter-cases.yaml

```diff
@@ -31,4 +31,50 @@
         curl -s -XPOST http://${sender_host}:${sender_9093}/sendBatchMetrics > /dev/null;
         sleep 10;
         swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=batch_test --instance-name=test-instance --service-name=test-service |yq e 'to_entries' -
-      expected: expected/metrics-has-value.yml
\ No newline at end of file
+      expected: expected/metrics-has-value.yml
+    # virtual cache
+    - query: |
+        swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=cache_read_resp_time  --service-id=R3VhdmFDYWNoZS1sb2NhbA==.0  | yq e 'to_entries' -
+      expected: expected/metrics-has-value0.yml
+    - query: |
+        swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=cache_read_sla  --service-id=R3VhdmFDYWNoZS1sb2NhbA==.0  | yq e 'to_entries' -
+      expected: expected/metrics-has-value.yml
+    - query: |
+        swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics linear --name=cache_read_cpm  --service-id=R3VhdmFDYWNoZS1sb2NhbA==.0  | yq e 'to_entries' -
+      expected: expected/metrics-has-value.yml
+    - query: |
+        swctl --display yaml --base-url=http://${oap_host}:${oap_12800}/graphql metrics multiple-linear --name=cache_read_percentile  --service-id=R3VhdmFDYWNoZS1sb2NhbA==.0  | yq e 'to_entries | with(.[] ; .value=(.value | to_entries))' -
+      expected: expected/metrics-has-value-percentile.yml
```

Comment:
To save cost and time in the CI, I think we don't have to test all metrics of the virtual cache.

You could choose one metric and sampled record.

Reply:
No, that is not testing about. What an e2e test is for checking the workflow of the target service. It costs time and money if you add all cases to verify. 
It is easy to say, let's add all cases, then one day, e2e would end in hours even over a day.